e2e_samples/fabric_dataops_sample/scripts/deploy_infrastructure.sh

```diff
@@ -207,6 +325,18 @@ deploy_terraform_resources "./infrastructure/terraform"
 echo "[Info] ############ Terraform resources deployed, setting up fabric bearer token ############"
 set_bearer_token
 
+echo "[Info] ############ ADLS Gen2 Cloud Connection Creation ############"
+
+if [[ -n "$adls_gen2_connection_id" ]]; then
```

Comment:
@promisinganuj i committed your suggested change... but then realize because `adls_gen2_connection_id` is never initialized before this line this will always be false, even if the connection does exist.

Reply:
@jose-perales , you are right, we are missing the following on line 330:

```bash
adls_gen2_connection_id=$(get_connection_id_by_name "$adls_gen2_connection_name")
```

Reply:
ahh... when i saw the line locally it was the cleanup script... adding to the deployment script now